cmake/setup_cross_compile.sh

```diff
@@ -0,0 +1,4 @@
+# Set the following variables as per tool installation on your system
+export CROSS_COMPILER_PATH=/workspace/shubham/ti/j7es_110523/workarea/gcc-arm-9.2-2019.12-x86_64-aarch64-none-linux-gnu
```

Comment:
Should these paths be so specific to your workarea?

Reply:
Updated the paths